scripts/gulp-tasks.js

```diff
@@ -0,0 +1,166 @@
+"use strict";
+
+/**
+ * This file contains the Gulp tasks for babel-standalone. Note that
+ * babel-standalone is compiled using Webpack, and performs its own Babel
+ * compilation of all the JavaScript files. This is because it targets web
+ * browsers, so more transforms are needed than the regular Babel builds that
+ * only target Node.js.
+ *
+ * The tasks in this file are designed to be reusable, so that they can be used
+ * to make standalone builds of other Babel plugins/presets (such as babel-minify)
+ */
+
+const path = require("path");
+const pump = require("pump");
+const rename = require("gulp-rename");
+const RootMostResolvePlugin = require("webpack-dependency-suite")
+  .RootMostResolvePlugin;
+const webpack = require("webpack");
+const webpackStream = require("webpack-stream");
+const uglify = require("gulp-uglify");
+
+const standalonePackagePlugins = {
+  babelPresetEnv: [
```

Comment:
Ideally this file should have no knowledge of the packages that it builds, as this gulp task may be used from other repos (for example, the babili / babel-minify repo). Could these extra options be passed in as an argument instead?